data/result-exporter/cne-exporter-commons/src/main/java/com/farao_community/farao/data/cne_exporter_commons/CneHelper.java

```diff
@@ -64,10 +67,26 @@ public Crac getCrac() {
         return crac;
     }
 
-    public StandardCracCreationContext getCracCreationContext() {
+    public CracCreationContext getCracCreationContext() {
         return cracCreationContext;
     }
 
+    public StandardCracCreationContext getStandardCracCreationContext() {
+        if (cracCreationContext instanceof StandardCracCreationContext) {
+            return (StandardCracCreationContext) cracCreationContext;
+        } else {
+            throw new FaraoException("Expected a StandardCracCreationContext.");
+        }
+    }
+
+    public CimCracCreationContext getCimCracCreationContext() {
+        if (cracCreationContext instanceof CimCracCreationContext) {
+            return (CimCracCreationContext) cracCreationContext;
+        } else {
+            throw new FaraoException("Expected a CimCracCreationContext.");
+        }
+    }
```

Comment:
I think I may have done this so the check (and cast) is only done in one place in the code rather than every time we need to get the cracCreationContext.
What do you prefer?